src/main/java/org/apache/accumulo/examples/sample/SampleExample.java

```diff
@@ -144,6 +143,7 @@ public static void main(String[] args) throws Exception {
 
       System.out.println("Scanning with new sampler configuration : ");
       print(scanner);
+      scanner.close();
```

Comment:
This should be in a try-with-resources block instead of an explicit close call.

Reply:
Addressed in [b31cf77](https://github.com/apache/accumulo-examples/pull/99/commits/b31cf77b86909f2e6ee239081a749a5dd6c2d0d5)